index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length ++
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items[pos]===undefined){
+      throw new Error('OutOfBounds');
+    }
+    return this.items[pos]
+  }
 
-  max() {}
+  max() {
+    if(this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }
 
-  min() {}
+    this.items.sort(function(a, b) {
```

Comment:
This is a very smart aproach! But you also can use the **Math.max()** inside a reduce method, take a look a it.